framework/source/class/qx/test/data/controller/FormWithArrayAndModel.js

```diff
@@ -0,0 +1,146 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2016 Martijn Evers, The Netherlands
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
```

Comment:
could you replace LGPL and EPG  with

`MIT: https://www.tldrlegal.com/l/mit`

since we are bound to go MIT rsn